pkg/testutil/db.go

```diff
@@ -212,32 +213,48 @@ func MustDo(t testing.TB, what string, err error) {
 }
 
 func NewBlockAdapterByType(t testing.TB, translator block.UploadIDTranslator, blockstoreType string) block.Adapter {
+	ctx := context.Background()
 	switch blockstoreType {
 	case block.BlockstoreTypeGS:
-		ctx := context.Background()
 		client, err := storage.NewClient(ctx)
 		if err != nil {
 			t.Fatal("Google Storage new client", err)
 		}
 		return gs.NewAdapter(client, gs.WithTranslator(translator))
 
 	case block.BlockstoreTypeS3:
+		var (
+			configLoadOpts []func(*aws_config.LoadOptions) error
+		)
+
+		configLoadOpts = append(configLoadOpts,
+			func(o *aws_config.LoadOptions) error {
+				o.LogConfigurationWarnings = aws.Bool(true)
+				return nil
+			})
```

Comment:
```suggestion
		configLoadOpts = append(configLoadOpts, aws_config.WithLogConfigurationWarnings(true))
```

too bad that they had a type function type and didn't use it for the setup - LoadOptionsFunc